src/System.CodeDom/tests/project.json

```diff
@@ -31,23 +31,6 @@
     "netstandard1.7": {}
   },
   "supports": {
-    "coreFx.Test.netcoreapp1.1": {
-      "netstandard1.7": [
-        "win7-x86",
-        "win7-x64",
-        "win10-arm64",
-        "osx.10.10-x64",
-        "centos.7-x64",
-        "debian.8-x64",
-        "rhel.7-x64",
-        "ubuntu.14.04-x64",
-        "ubuntu.16.04-x64",
-        "ubuntu.16.10-x64",
-        "fedora.23-x64",
-        "linux-x64",
-        "opensuse.13.2-x64",
-        "opensuse.42.1-x64"
-      ]
-    }
+    "coreFx.Test.netcoreapp1.1": {}
```

Comment:
@stephentoub in order to run these on desktop you will need this support clause. After you do that you have too possible ways to run:
```
msbuild System.CodeDom.Tests.csproj /p:OSGroup=Windows_NT /p:TestTFM=net463
or
msbuild System.CodeDom.Tests.builds /p:FilterToTestTFM=net463
```

For better or worse when building the builds file we build all combinations but only test the what is set in FilterToTestTFM which you can pass directory or it defaults to https://github.com/dotnet/corefx/pull/12241/files#diff-0b192804a6349e8c26d2b027afbd89a2R504.